### DISCUSSION TITLE
BIG NUMBER GROUP BY

### DISCUSSION THREAD

**Discussion by AHZenan** [apache/superset#19378] [Q&A / Help]
## Screenshot

![image](https://user-images.githubusercontent.com/81419019/160176783-f92693c8-bf30-406d-8e93-0765393d9cb1.png)
![image](https://user-images.githubusercontent.com/81419019/160177044-bf6473e3-3f7c-47b4-b47c-d91d5af4e216.png)
![image](https://user-images.githubusercontent.com/81419019/160177100-cdd85b63-dbf6-4156-80ee-9d57e0cfa29b.png)

## Description

I want to use the big number to display a single sum  ( in the example I have used no of buildings) however i want the filter to be applied on the dashboard, where if the user chooses "All", the big number would show 24, if the user chooses "Neath" , big number would show 10 and if the user chooses "Carnego" big number would be 14. How can I achieve this ?

**Comment by srinify** [comment]
Hey Zenan! Moving to discussions since issues are for bugs primarily :)
